app/assets/stylesheets/application.css.scss

```diff
@@ -16,11 +16,11 @@
  */
  @font-face {
 	font-family: 'icomoon';
-	src:font-url('icomoon.eot?-adig3h');
-	src:font-url('icomoon.eot?#iefix-adig3h') format('embedded-opentype'),
-		font-url('icomoon.ttf?-adig3h') format('truetype'),
-		font-url('icomoon.woff?-adig3h') format('woff'),
-		font-url('icomoon.svg?-adig3h#icomoon') format('svg');
+	src:font-url('icomoon.eot?vlkd6h');
+	src:font-url('icomoon.eot?#iefixvlkd6h') format('embedded-opentype'),
```

Comment:
Colon after property should be followed by one space<br>Prefer double-quoted strings